app/views/thredded/shared/_topic_nav.html.erb

```diff
@@ -2,19 +2,21 @@
   <%= yield :thredded_breadcrumbs %>
 
   <ul class="thredded--topic-navigation">
-    <% if messageboard.try(:persisted?) %>
+    <% if messageboard.try(:persisted?) -%>
```

Comment:
I know what persisted is checking for, I mean how would a user get here?

Makes sense from a new perspective since this is a shared nav.